com.remnantsoftheprecursors.ROTP.metainfo.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.remnantsoftheprecursors.ROTP</id>
+  <name>Remnants of the Precursors</name>
+  <metadata_license> CC-BY-NC-ND-4.0</metadata_license>
```

Comment:
this is not an accepted license for metadata. `CC0-1.0` is usually preferred.

Just to clarify this is not the game asset license. This is the license of THIS file.

Reply:
ok fixing asap